era_5g_relay_network_application/worker_service.py

```diff
@@ -63,12 +102,18 @@ def run(self) -> None:
             try:
                 sid, data = self.requests_queue.get(block=True, timeout=1)
 
-                future: Future = self.client.call_async(populate_instance(data, self.inst))
-                future.add_done_callback(partial(self.callback_future, sid=sid))
+                if self.action_service_variant == ActionServiceVariant.ACTION_SEND_GOAL:
+                    goal_uuid = data["goal_id"]["uuid"]
+                    self.action_subscribers.set_sid_for_goal_id(goal_uuid, sid)  # type: ignore
```

Comment:
mypy seems to be unable to infer that `self.action_subscribers` will not be `None` here and throws an error. Adding `assert self.action_subscribers is not None` helps, but we could use `# type: ignore` instead of the meaningless assert...? What do you consider better?

Reply:
After more discussion, adding `assert self.action_subscribers is not None` was selected as the preferred solution.

Reply:
@idobes Maybe you forgot to add those changes to the commit?

Reply:
Fixed :)